mathics/builtin/no_meaning/infix_extra.py

```diff
@@ -44,20 +43,20 @@ class DirectedEdge(InfixOperator):
 
     attributes = A_NO_ATTRIBUTES
     default_formats = False  # Don't use any default format rules. Instead, see belo.
-    summary_text = 'DirectedEdge infix operator "→"'
+    summary_text = 'DirectedEdge infix operator "->"'
 
 
 class UndirectedEdge(InfixOperator):
-    """
+    r"""
     <url>
     :WML link:
     https://reference.wolfram.com/language/ref/UndirectedEdge.html</url>
 
     <dl>
       <dt>'UndrectedEdge[$x$, $y$, ...]'
-      <dd>displays $x$ ↔ $y$ ↔ ...
+      <dd>displays $x <-> y$ ...
```

Comment:
A couple things to note. 

First, although the docsystem might not be able to handle the unicode, nevertheless `UndirectedEdge[]` does not display `<->`: 

```
MATHICS_CHARACTER_ENCODING="ASCII" mathics -e 'UndirectedEdge[x, y]'
x ↔ y
```

Maybe one day it might. 

Second while` $x <-> y$` is correct TeX/LaTeX formatting, it is not the way homegrown (to be replaced) Mathics works. `$x$ <-> $y$ is correct I think.

Reply:
Sigh.